src/traversing/findFilter.js

```diff
@@ -25,14 +25,17 @@ function winnow( elements, qualifier, not ) {
 
 	}
 
-	if ( typeof qualifier === "string" ) {
-		if ( risSimple.test( qualifier ) ) {
-			return jQuery.filter( qualifier, elements, not );
-		}
+	if ( typeof qualifier !== "string" ) {
```

Comment:
Maybe couple comments on what's for what? Not immediately obvious right now, although it sounds like a bonus round